apps/noaabuoy/noaa_buoy.star

```diff
@@ -0,0 +1,147 @@
+"""
+Applet: NOAA Buoy
+Summary: Display buoy wave data
+Description: Display swell data for user specified buoy. Find buoy_id's here : https://www.ndbc.noaa.gov/obs.shtml Buoy must have height,period,direction to display correctly
+Author: tavdog
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+
+def fetch_data(buoy_id):
+    url = "http://wildc.net/wind/noaa_buoy_api.pl?buoy_id=%s" % buoy_id
+    resp = http.get(url)
+    if resp.status_code != 200:
+        fail("request failed with status %d", resp.status_code)
```

Comment:
If I'm correct that this is self hosted, we should reply with a message to the user letting them know the server is down rather then failing the script